### ISSUE TITLE
--show-profile crashes

### ISSUE THREAD

**Issue by Deimos** [PyCQA/prospector#279]
Trying to run `prospector --show-profile` crashes with an `AttributeError`:

```
Traceback (most recent call last):
  File "/opt/venvs/tildes/bin/prospector", line 11, in <module>
    load_entry_point('prospector==1.1.2', 'console_scripts', 'prospector')()
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/run.py", line 174, in main
    prospector.print_messages()
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/run.py", line 134, in print_messages
    self.write_to(formatter, sys.stdout)
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/run.py", line 144, in write_to
    profile=self.config.show_profile
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/formatters/text.py", line 106, in render
    output.append(self.render_profile())
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/formatters/text.py", line 96, in render_profile
    self.profile.as_yaml().strip()
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/profiles/profile.py", line 115, in as_yaml
    return yaml.safe_dump(self.as_dict())
  File "/opt/venvs/tildes/lib/python3.6/site-packages/prospector/profiles/profile.py", line 96, in as_dict
    'output-file': self.output_file,
AttributeError: 'ProspectorProfile' object has no attribute 'output_file'
```

**Comment by chocoelho** [comment]
Will check that today

**Comment by chocoelho** [comment]
Fixed in `1.1.3`. Sorry for the delay, it's been crazy over here.
